### PR TITLE
docs: add IPMI info, fix links, update environment

### PR DESCRIPTION
**Pull request by smira**
Update environment kernel args and links to the latest Talos release,
add IPMI info, fix random broken links.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: docs/website/content/docs/v0.1/Configuration/servers.md [hunk 1]
```diff
@@ -85,3 +85,27 @@ You should never change an accepted `Server` to be _not_ accepted while it is in
...
+
+## IPMI
+
+Sidero can use IPMI information to control `Server` power state, reboot servers and set boot order.
+
+IMPI connection information can be set in the `Server` spec after initial registration:
+
+```yaml
+apiVersion: metal.sidero.dev/v1alpha1
+kind: ServerClass
```
Comment (nberlee) [inline docs/website/content/docs/v0.1/Configuration/servers.md:97]: I think you mean `kind: Server` here, as `ServerClass` does not have `bmc` according to `kubectl explain serverclass.spec`

Comment (andrewrynhard) [reply docs/website/content/docs/v0.1/Configuration/servers.md:97]: Yep, this is correct. Good catch.

Comment (smira) [reply docs/website/content/docs/v0.1/Configuration/servers.md:97]: thanks for catching that!

**Review by andrewrynhard — APPROVED** [review]

File: sfyra/pkg/capi/cluster.go [hunk 2]
```diff
@@ -89,6 +89,14 @@ func NewCluster(ctx context.Context, metalClient runtimeclient.Reader, clusterNa
...
 
 		for _, machine := range machines.Items {
+			if !machine.DeletionTimestamp.IsZero() {
+				continue
+			}
+
+			if v1alpha3.MachinePhase(machine.Status.Phase) != v1alpha3.MachinePhaseRunning {
+				continue
+			}
+
```
Comment (andrewrynhard) [inline sfyra/pkg/capi/cluster.go:99]: This the fix for the worker scaling flakes?

Comment (smira) [reply sfyra/pkg/capi/cluster.go:99]: yes, yes, and yes!

Comment (andrewrynhard) [reply sfyra/pkg/capi/cluster.go:99]: Nice 😄

Comment (smira) [reply sfyra/pkg/capi/cluster.go:99]: I got fed up restarting the build, so I went ahead and fixed it hopefully.

Basically machine was marked as deleted, replicas in the status of machinedeployment is accurate, but we picked up stale info by looking into deleted machine

**Pull request merged**